test-framework/ui/src/main/java/org/keycloak/testframework/ui/page/ErrorPage.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.testframework.ui.page;
+
+import org.keycloak.testframework.ui.util.PageUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
+ */
+public class ErrorPage extends AbstractPage {
+
+    @FindBy(className = "instruction")
+    private WebElement errorMessage;
+
+    @FindBy(id = "backToApplication")
+    private WebElement backToApplicationLink;
+
+    public ErrorPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String getError() {
+        return errorMessage.getText();
+    }
+
+    public void clickBackToApplication() {
+        backToApplicationLink.click();
+    }
+
+    public String getBackToApplicationLink() {
+        if (backToApplicationLink == null) {
+            return null;
+        } else {
+            return backToApplicationLink.getAttribute("href");
+        }
+    }
+
+    @Override
+    public String getExpectedPageId() {
+        return "error";
+    }
+
+    @Override
+    public boolean isCurrent() {
+        return PageUtils.getPageTitle(driver) != null && PageUtils.getPageTitle(driver).equals("We are sorry...");
+    }
```

Comment:
See comment in AbstractPage for this page and all others. No need to implement isCurrent or assertCurrent or whatever, just getExpectedPageId